README.md

```diff
@@ -0,0 +1,70 @@
+# GPS Visualiser
```

Comment:
If you want somebody else to use your code, you will need to add a LICENSE file (either MIT or GNU). You can copy and paste this one: https://github.com/sanger-bentley-group/GPSC_pipeline_nf/blob/master/LICENSE

Reply:
Will do!